README.md

```diff
@@ -0,0 +1,64 @@
+# Vite 2.0
+===========
+
+Vite (с французского "быстрый", произносится /вит/) - это инструмент сборки, созданный для обеспечения быстрого и бережливого (lean) процесса разработки современных веб-проектов.
+
+- [Ссылка на докуметацию](https://vite-docs-ru.vercel.app/)
+
+## О нас
+Русскоязычное сообщество, помогающее развавать новые технологии.
+
+## Установка зависимостей
+
+```bash
+yarn install
+```
+
+### Запуск приложения в режиме разработки (build mode)
+
+```bash
+npm run docs 
```

Comment:
If you started to use `Yarn` in first step, why on the next steps you use `npm`?

Reply:
> If you started to use `Yarn` in first step, why on the next steps you use `npm`?

to avoid on win 8.1 issues